tests/unit/mpool/mblock_test.c

```diff
@@ -229,10 +210,7 @@ MTF_DEFINE_UTEST_PREPOST(mblock_test, mblock_abc, mpool_test_pre, mpool_test_pos
     ASSERT_EQ(props.mpr_alloc_cap, 0);
 
     err = mpool_mblock_delete(mp, mbid);
-    ASSERT_EQ(0, err);
-
-    err = mpool_mblock_abort(mp, mbid);
```

Comment:
Any reason you had removed this? I suppose that calling mpool_mblock_delete() for the second time should return ENOENT.

Reply:
Never mind, I see that this case is already tested few lines above.